geometry_msgs/Vector3 sip1
geometry_msgs/Vector3 sip2
geometry_msgs/Vector3 sip3
geometry_msgs/Vector3 sip4
geometry_msgs/Vector3 sip5
geometry_msgs/Vector3 sip6
std_msgs/Bool takeoff_flag
